ros_ws/src/capstone_interfaces/msg/StateObject.msg

```diff
@@ -4,5 +4,4 @@ string location
 float32 x
 float32 y
 float32 z
-string task_when_seen
```

Comment:
This change came out of the call last week when talking about what state management expected as inputs. It sounded to me like the consensus was that it wasn't necessary to keep track of. I think the effect of having it would skew the similarity scores towards objects that have been seen while looking for the object asked about, which I don't think particularly helps or hurts us.

Reply:
Sounds good to me! 👍🏻